Makefile

```diff
@@ -18,11 +18,9 @@ APP_VERSION :=
 HACK_DIR ?= hack
 
 # Get a list of all binaries to be built
-CMDS := $(shell find ./cmd/ -maxdepth 1 -type d -exec basename {} \; | grep -v cmd)
-
-.PHONY: help build verify push $(CMDS) images images_push \
-	verify_deps verify_chart
+CMDS := $(shell find ./cmd/ -maxdepth 1 -mindepth 1 -type d -exec basename {} \;)
```

Comment:
note for reviewers: we can avoid piping here and improve performance very, very slightly. below is a temporary target I made, showing that the output is the same.

```make
OLD_CMDS := $(shell find ./cmd/ -maxdepth 1 -type d -exec basename {} \; | grep -v cmd)
CMDS := $(shell find ./cmd/ -maxdepth 1 -mindepth 1 -type d -exec basename {} \;)

.PHONY: cmds
cmds:
	@echo $(CMDS)
	@echo $(OLD_CMDS)
```

```bash
$ make cmds
webhook controller acmesolver ctl cainjector
webhook controller acmesolver ctl cainjector
```

Reply:
Double checked this works on MacOS